src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/AttendanceChangeValues.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model
+
+
+enum class AttendanceChangeValues {
```

Comment:
What I'd really like to do with the whole attendance model is remove the absent reason and attended boolean combination and have an overall attendance status much like this enum.

Reply:
That would make it less brittle 👍 I also like the companion object thingie (I may have just had a course on them) for AbsentReason, much neater!